src/scene/gsplat/gsplat-resolve-sh.js

```diff
@@ -0,0 +1,349 @@
+import { Vec3 } from '../../core/math/vec3.js';
+import { Mat4 } from '../../core/math/mat4.js';
+import { BlendState } from '../../platform/graphics/blend-state.js';
+import {
+    CULLFACE_NONE,
+    PIXELFORMAT_RGBA8,
+    SEMANTIC_POSITION
+} from '../../platform/graphics/constants.js';
+import { DepthState } from '../../platform/graphics/depth-state.js';
+import { RenderTarget } from '../../platform/graphics/render-target.js';
+import { ShaderUtils } from '../shader-lib/shader-utils.js';
+import { ShaderChunks } from '../shader-lib/shader-chunks.js';
+import { Vec2 } from '../../core/math/vec2.js';
+import { RenderPass } from '../../platform/graphics/render-pass.js';
+import { QuadRender } from '../graphics/quad-render.js';
+
+const vertexGLSL = /* glsl */`
+    attribute vec2 vertex_position;
+    void main(void) {
+        gl_Position = vec4(vertex_position, 0.0, 1.0);
+    }
+`;
+
+const fragmentGLSL = /* glsl */`
+    #if SH_BANDS == 1
+        #define SH_COEFFS 3
+    #elif SH_BANDS == 2
+        #define SH_COEFFS 8
+    #elif SH_BANDS == 3
+        #define SH_COEFFS 15
+    #endif
+
+    #define SH_C1 0.4886025119029199f
+
+    #if SH_BANDS > 1
+        #define SH_C2_0 1.0925484305920792f
+        #define SH_C2_1 -1.0925484305920792f
+        #define SH_C2_2 0.31539156525252005f
+        #define SH_C2_3 -1.0925484305920792f
+        #define SH_C2_4 0.5462742152960396f
+    #endif
+
+    #if SH_BANDS > 2
+        #define SH_C3_0 -0.5900435899266435f
+        #define SH_C3_1 2.890611442640554f
+        #define SH_C3_2 -0.4570457994644658f
+        #define SH_C3_3 0.3731763325901154f
+        #define SH_C3_4 -0.4570457994644658f
+        #define SH_C3_5 1.445305721320277f
+        #define SH_C3_6 -0.5900435899266435f
+    #endif
+
+    // see https://github.com/graphdeco-inria/gaussian-splatting/blob/main/utils/sh_utils.py
+    vec3 evalSH(in vec3 sh[SH_COEFFS], in vec3 dir) {
```

Comment:
this looks like a copy & paste from another chunk. Should we extract that function to a new chunk and include it in both places.